opentelemetry-otlp/examples/basic-otlp-http/src/main.rs

```diff
@@ -63,8 +63,18 @@ fn init_metrics() -> Result<opentelemetry_sdk::metrics::SdkMeterProvider, Metric
         .with_endpoint("http://localhost:4318/v1/metrics")
         .build()?;
 
+    #[cfg(feature = "experimental_metrics_periodicreader_with_async_runtime")]
+    let reader =
+        opentelemetry_sdk::metrics::periodic_reader_with_async_runtime::PeriodicReader::builder(
+            exporter,
+            runtime::Tokio,
+        )
+        .build();
+    #[cfg(not(feature = "experimental_metrics_periodicreader_with_async_runtime"))]
```

Comment:
Let's add a comment saying for now this would only work when called from a regular main method.